# HG changeset patch
# Parent b55b3978eccc285fe50f2e95231f40f28abfa107
# User Frédéric Wang <fred.wang@free.fr>
MathML Operator Dictionary: fix conflicts for entries becoming accent (bug 534970). r=karlt

diff --git a/layout/mathml/mathfont.properties b/layout/mathml/mathfont.properties
--- a/layout/mathml/mathfont.properties
+++ b/layout/mathml/mathfont.properties
@@ -120,17 +120,17 @@ mathvariant.monospace = A\ud835\ude70B\u
 # Each entry lists the attributes of the operator, using its Unicode format.
 
 operator.\u0021.postfix = lspace:1 rspace:0 # !
 operator.\u0021\u0021.postfix = lspace:1 rspace:0 # !!
 operator.\u0021\u003D.infix = lspace:4 rspace:4 # !=
 operator.\u0025.infix = lspace:3 rspace:3 # percent sign
 operator.\u0026.postfix = lspace:0 rspace:0 # &amp;
 operator.\u0026\u0026.infix = lspace:4 rspace:4 # &amp;&amp;
-operator.\u0027.postfix = lspace:2 rspace:0 # '
+operator.\u0027.postfix = lspace:0 rspace:0 accent # '
 operator.\u0028.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # (
 operator.\u0029.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # )
 operator.\u002A.infix = lspace:3 rspace:3 # *
 operator.\u002A\u002A.infix = lspace:1 rspace:1 # **
 operator.\u002A\u003D.infix = lspace:4 rspace:4 # *=
 operator.\u002B.infix = lspace:4 rspace:4 # +
 operator.\u002B.prefix = lspace:0 rspace:1 # +
 operator.\u002B\u002B.postfix = lspace:0 rspace:0 # ++
@@ -218,124 +218,124 @@ operator.\u203E.postfix = lspace:0 rspac
 operator.\u2061.infix = lspace:0 rspace:0 # &ApplyFunction;
 operator.\u2062.infix = lspace:0 rspace:0 # &InvisibleTimes;
 operator.\u2063.infix = lspace:0 rspace:0 separator # &InvisibleComma;
 operator.\u2064.infix = lspace:0 rspace:0 # invisible plus
 operator.\u20DB.postfix = lspace:0 rspace:0 accent # &TripleDot;
 operator.\u20DC.postfix = lspace:0 rspace:0 accent # combining four dots above
 operator.\u2145.prefix = lspace:2 rspace:1 # &CapitalDifferentialD;
 operator.\u2146.prefix = lspace:2 rspace:0 # &DifferentialD;
-operator.\u2190.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftArrow;
+operator.\u2190.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &LeftArrow;
 operator.\u2191.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpArrow;
-operator.\u2192.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightArrow;
+operator.\u2192.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &RightArrow;
 operator.\u2193.infix = lspace:5 rspace:5 stretchy direction:vertical # &DownArrow;
-operator.\u2194.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftRightArrow;
+operator.\u2194.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &LeftRightArrow;
 operator.\u2195.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpDownArrow;
 operator.\u2196.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpperLeftArrow;
 operator.\u2197.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpperRightArrow;
 operator.\u2198.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LowerRightArrow;
 operator.\u2199.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LowerLeftArrow;
 operator.\u219A.infix = lspace:5 rspace:5 accent # leftwards arrow with stroke
 operator.\u219B.infix = lspace:5 rspace:5 accent # rightwards arrow with stroke
-operator.\u219C.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards wave arrow
-operator.\u219D.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards wave arrow
-operator.\u219E.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards two headed arrow
-operator.\u219F.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards two headed arrow
-operator.\u21A0.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards two headed arrow
+operator.\u219C.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # leftwards wave arrow
+operator.\u219D.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # rightwards wave arrow
+operator.\u219E.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # leftwards two headed arrow
+operator.\u219F.infix = lspace:5 rspace:5 stretchy accent direction:vertical # upwards two headed arrow
+operator.\u21A0.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # rightwards two headed arrow
 operator.\u21A1.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards two headed arrow
-operator.\u21A2.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards arrow with tail
-operator.\u21A3.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards arrow with tail
-operator.\u21A4.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftTeeArrow;
+operator.\u21A2.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # leftwards arrow with tail
+operator.\u21A3.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # rightwards arrow with tail
+operator.\u21A4.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &LeftTeeArrow;
 operator.\u21A5.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpTeeArrow;
-operator.\u21A6.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightTeeArrow;
+operator.\u21A6.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &RightTeeArrow;
 operator.\u21A7.infix = lspace:5 rspace:5 stretchy direction:vertical # &DownTeeArrow;
 operator.\u21A8.infix = lspace:5 rspace:5 stretchy direction:vertical # up down arrow with base
-operator.\u21A9.infix = lspace:5 rspace:5 stretchy direction:horizontal # &hookleftarrow; &larrhk;
-operator.\u21AA.infix = lspace:5 rspace:5 stretchy direction:horizontal # &hookrightarrow; &rarrhk;
-operator.\u21AB.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards arrow with loop
-operator.\u21AC.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards arrow with loop
-operator.\u21AD.infix = lspace:5 rspace:5 stretchy direction:horizontal # left right wave arrow
+operator.\u21A9.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &hookleftarrow; &larrhk;
+operator.\u21AA.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &hookrightarrow; &rarrhk;
+operator.\u21AB.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # leftwards arrow with loop
+operator.\u21AC.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # rightwards arrow with loop
+operator.\u21AD.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # left right wave arrow
 operator.\u21AE.infix = lspace:5 rspace:5 accent # left right arrow with stroke
 operator.\u21AF.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards zigzag arrow
 operator.\u21B0.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards arrow with tip leftwards
 operator.\u21B1.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards arrow with tip rightwards
 operator.\u21B2.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards arrow with tip leftwards
 operator.\u21B3.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards arrow with tip rightwards
 operator.\u21B4.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards arrow with corner downwards
 operator.\u21B5.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards arrow with corner leftwards
 operator.\u21B6.infix = lspace:5 rspace:5 accent # anticlockwise top semicircle arrow
 operator.\u21B7.infix = lspace:5 rspace:5 accent # clockwise top semicircle arrow
 operator.\u21B8.infix = lspace:5 rspace:5 # north west arrow to long bar
-operator.\u21B9.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards arrow to bar over rightwards arrow to bar
+operator.\u21B9.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # leftwards arrow to bar over rightwards arrow to bar
 operator.\u21BA.infix = lspace:5 rspace:5 # anticlockwise open circle arrow
 operator.\u21BB.infix = lspace:5 rspace:5 # clockwise open circle arrow
-operator.\u21BC.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftVector;
-operator.\u21BD.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownLeftVector;
+operator.\u21BC.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &LeftVector;
+operator.\u21BD.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &DownLeftVector;
 operator.\u21BE.infix = lspace:5 rspace:5 stretchy direction:vertical # &RightUpVector;
 operator.\u21BF.infix = lspace:5 rspace:5 stretchy direction:vertical # &LeftUpVector;
-operator.\u21C0.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightVector;
-operator.\u21C1.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownRightVector;
+operator.\u21C0.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &RightVector;
+operator.\u21C1.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &DownRightVector;
 operator.\u21C2.infix = lspace:5 rspace:5 stretchy direction:vertical # &RightDownVector;
 operator.\u21C3.infix = lspace:5 rspace:5 stretchy direction:vertical # &LeftDownVector;
-operator.\u21C4.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightArrowLeftArrow;
+operator.\u21C4.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &RightArrowLeftArrow;
 operator.\u21C5.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpArrowDownArrow;
-operator.\u21C6.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftArrowRightArrow;
-operator.\u21C7.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards paired arrows
+operator.\u21C6.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &LeftArrowRightArrow;
+operator.\u21C7.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # leftwards paired arrows
 operator.\u21C8.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards paired arrows
-operator.\u21C9.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards paired arrows
+operator.\u21C9.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # rightwards paired arrows
 operator.\u21CA.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards paired arrows
-operator.\u21CB.infix = lspace:5 rspace:5 stretchy direction:horizontal # &ReverseEquilibrium;
-operator.\u21CC.infix = lspace:5 rspace:5 stretchy direction:horizontal # &Equilibrium;
+operator.\u21CB.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &ReverseEquilibrium;
+operator.\u21CC.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &Equilibrium;
 operator.\u21CD.infix = lspace:5 rspace:5 accent # leftwards double arrow with stroke
 operator.\u21CE.infix = lspace:5 rspace:5 accent # left right double arrow with stroke
 operator.\u21CF.infix = lspace:5 rspace:5 accent # rightwards double arrow with stroke
-operator.\u21D0.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DoubleLeftArrow;
+operator.\u21D0.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &DoubleLeftArrow;
 operator.\u21D1.infix = lspace:5 rspace:5 stretchy direction:vertical # &DoubleUpArrow;
-operator.\u21D2.infix = lspace:5 rspace:5 stretchy direction:horizontal # &Implies; &DoubleRightArrow;
+operator.\u21D2.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &Implies; &DoubleRightArrow;
 operator.\u21D3.infix = lspace:5 rspace:5 stretchy direction:vertical # &DoubleDownArrow;
-operator.\u21D4.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DoubleLeftRightArrow;
+operator.\u21D4.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &DoubleLeftRightArrow;
 operator.\u21D5.infix = lspace:5 rspace:5 stretchy direction:vertical # &DoubleUpDownArrow;
-operator.\u21DA.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards triple arrow
-operator.\u21DB.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards triple arrow
-operator.\u21DC.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards squiggle arrow
-operator.\u21DD.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards squiggle arrow
+operator.\u21DA.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # leftwards triple arrow
+operator.\u21DB.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # rightwards triple arrow
+operator.\u21DC.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # leftwards squiggle arrow
+operator.\u21DD.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # rightwards squiggle arrow
 operator.\u21DE.infix = lspace:5 rspace:5 # upwards arrow with double stroke
 operator.\u21DF.infix = lspace:5 rspace:5 # downwards arrow with double stroke
-operator.\u21E0.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards dashed arrow
+operator.\u21E0.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # leftwards dashed arrow
 operator.\u21E1.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards dashed arrow
-operator.\u21E2.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards dashed arrow
+operator.\u21E2.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # rightwards dashed arrow
 operator.\u21E3.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards dashed arrow
-operator.\u21E4.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftArrowBar;
-operator.\u21E5.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightArrowBar;
-operator.\u21E6.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards white arrow
+operator.\u21E4.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &LeftArrowBar;
+operator.\u21E5.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &RightArrowBar;
+operator.\u21E6.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # leftwards white arrow
 operator.\u21E7.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards white arrow
-operator.\u21E8.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards white arrow
+operator.\u21E8.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # rightwards white arrow
 operator.\u21E9.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards white arrow
 operator.\u21EA.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards white arrow from bar
 operator.\u21EB.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards white arrow on pedestal
 operator.\u21EC.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards white arrow on pedestal with horizontal bar
 operator.\u21ED.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards white arrow on pedestal with vertical bar
 operator.\u21EE.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards white double arrow
 operator.\u21EF.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards white double arrow on pedestal
-operator.\u21F0.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards white arrow from wall
+operator.\u21F0.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # rightwards white arrow from wall
 operator.\u21F1.infix = lspace:5 rspace:5 # north west arrow to corner
 operator.\u21F2.infix = lspace:5 rspace:5 # south east arrow to corner
 operator.\u21F3.infix = lspace:5 rspace:5 stretchy direction:vertical # up down white arrow
 operator.\u21F4.infix = lspace:5 rspace:5 accent # right arrow with small circle
 operator.\u21F5.infix = lspace:5 rspace:5 stretchy direction:vertical # &DownArrowUpArrow;
-operator.\u21F6.infix = lspace:5 rspace:5 stretchy direction:horizontal # three rightwards arrows
+operator.\u21F6.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # three rightwards arrows
 operator.\u21F7.infix = lspace:5 rspace:5 accent # leftwards arrow with vertical stroke
 operator.\u21F8.infix = lspace:5 rspace:5 accent # rightwards arrow with vertical stroke
 operator.\u21F9.infix = lspace:5 rspace:5 accent # left right arrow with vertical stroke
 operator.\u21FA.infix = lspace:5 rspace:5 accent # leftwards arrow with double vertical stroke
 operator.\u21FB.infix = lspace:5 rspace:5 accent # rightwards arrow with double vertical stroke
 operator.\u21FC.infix = lspace:5 rspace:5 accent # left right arrow with double vertical stroke
-operator.\u21FD.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards open-headed arrow
-operator.\u21FE.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards open-headed arrow
-operator.\u21FF.infix = lspace:5 rspace:5 stretchy direction:horizontal # left right open-headed arrow
+operator.\u21FD.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # leftwards open-headed arrow
+operator.\u21FE.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # rightwards open-headed arrow
+operator.\u21FF.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # left right open-headed arrow
 operator.\u2200.prefix = lspace:2 rspace:1 # &ForAll;
 operator.\u2201.infix = lspace:1 rspace:2 # complement
 operator.\u2202.prefix = lspace:2 rspace:1 # &PartialD;
 operator.\u2203.prefix = lspace:2 rspace:1 # &Exists;
 operator.\u2204.prefix = lspace:2 rspace:1 # &NotExists;
 operator.\u2206.infix = lspace:3 rspace:3 # increment
 operator.\u2207.prefix = lspace:2 rspace:1 # &Del;
 operator.\u2208.infix = lspace:5 rspace:5 # &Element;
@@ -658,22 +658,22 @@ operator.\u27E7.postfix = lspace:0 rspac
 operator.\u27E8.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &LeftAngleBracket;
 operator.\u27E9.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &RightAngleBracket;
 operator.\u27EA.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # mathematical left double angle bracket
 operator.\u27EB.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # mathematical right double angle bracket
 operator.\u27EC.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # mathematical left white tortoise shell bracket
 operator.\u27ED.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # mathematical right white tortoise shell bracket
 operator.\u27EE.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # mathematical left flattened parenthesis
 operator.\u27EF.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # mathematical right flattened parenthesis
-operator.\u27F5.infix = lspace:2 rspace:2 stretchy direction:horizontal # &LongLeftArrow;
-operator.\u27F6.infix = lspace:2 rspace:2 stretchy direction:horizontal # &LongRightArrow;
-operator.\u27F7.infix = lspace:2 rspace:2 stretchy direction:horizontal # &LongLeftRightArrow;
-operator.\u27F8.infix = lspace:2 rspace:2 stretchy direction:horizontal # &DoubleLongLeftArrow;
-operator.\u27F9.infix = lspace:2 rspace:2 stretchy direction:horizontal # &DoubleLongRightArrow;
-operator.\u27FA.infix = lspace:2 rspace:2 stretchy direction:horizontal # &DoubleLongLeftRightArrow;
+operator.\u27F5.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &LongLeftArrow;
+operator.\u27F6.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &LongRightArrow;
+operator.\u27F7.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &LongLeftRightArrow;
+operator.\u27F8.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &DoubleLongLeftArrow;
+operator.\u27F9.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &DoubleLongRightArrow;
+operator.\u27FA.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &DoubleLongLeftRightArrow;
 operator.\u27FB.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # long leftwards arrow from bar
 operator.\u27FC.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # long rightwards arrow from bar
 operator.\u27FD.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # long leftwards double arrow from bar
 operator.\u27FE.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # long rightwards double arrow from bar
 operator.\u27FF.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # long rightwards squiggle arrow
 operator.\u2900.infix = lspace:5 rspace:5 accent # rightwards two-headed arrow with vertical stroke
 operator.\u2901.infix = lspace:5 rspace:5 accent # rightwards two-headed arrow with double vertical stroke
 operator.\u2902.infix = lspace:5 rspace:5 accent # leftwards double arrow with vertical stroke
@@ -681,21 +681,21 @@ operator.\u2903.infix = lspace:5 rspace:
 operator.\u2904.infix = lspace:5 rspace:5 accent # left right double arrow with vertical stroke
 operator.\u2905.infix = lspace:5 rspace:5 accent # rightwards two-headed arrow from bar
 operator.\u2906.infix = lspace:5 rspace:5 accent # leftwards double arrow from bar
 operator.\u2907.infix = lspace:5 rspace:5 accent # rightwards double arrow from bar
 operator.\u2908.infix = lspace:5 rspace:5 # downwards arrow with horizontal stroke
 operator.\u2909.infix = lspace:5 rspace:5 # upwards arrow with horizontal stroke
 operator.\u290A.infix = lspace:5 rspace:5 stretchy direction:vertical # upwards triple arrow
 operator.\u290B.infix = lspace:5 rspace:5 stretchy direction:vertical # downwards triple arrow
-operator.\u290C.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards double dash arrow
-operator.\u290D.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards double dash arrow
-operator.\u290E.infix = lspace:5 rspace:5 stretchy direction:horizontal # leftwards triple dash arrow
-operator.\u290F.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards triple dash arrow
-operator.\u2910.infix = lspace:5 rspace:5 stretchy direction:horizontal # rightwards two-headed triple dash arrow
+operator.\u290C.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # leftwards double dash arrow
+operator.\u290D.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # rightwards double dash arrow
+operator.\u290E.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # leftwards triple dash arrow
+operator.\u290F.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # rightwards triple dash arrow
+operator.\u2910.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # rightwards two-headed triple dash arrow
 operator.\u2911.infix = lspace:5 rspace:5 accent # rightwards arrow with dotted stem
 operator.\u2912.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpArrowBar;
 operator.\u2913.infix = lspace:5 rspace:5 stretchy direction:vertical # &DownArrowBar;
 operator.\u2914.infix = lspace:5 rspace:5 accent # rightwards arrow with tail with vertical stroke
 operator.\u2915.infix = lspace:5 rspace:5 accent # rightwards arrow with tail with double vertical stroke
 operator.\u2916.infix = lspace:5 rspace:5 accent # rightwards two-headed arrow with tail
 operator.\u2917.infix = lspace:5 rspace:5 accent # rightwards two-headed arrow with tail with vertical stroke
 operator.\u2918.infix = lspace:5 rspace:5 accent # rightwards two-headed arrow with tail with double vertical stroke
@@ -745,51 +745,51 @@ operator.\u2945.infix = lspace:5 rspace:
 operator.\u2946.infix = lspace:5 rspace:5 accent # leftwards arrow with plus below
 operator.\u2947.infix = lspace:5 rspace:5 accent # rightwards arrow through x
 operator.\u2948.infix = lspace:5 rspace:5 accent # left right arrow through small circle
 operator.\u2949.infix = lspace:5 rspace:5 # upwards two-headed arrow from small circle
 operator.\u294A.infix = lspace:5 rspace:5 accent # left barb up right barb down harpoon
 operator.\u294B.infix = lspace:5 rspace:5 accent # left barb down right barb up harpoon
 operator.\u294C.infix = lspace:5 rspace:5 # up barb right down barb left harpoon
 operator.\u294D.infix = lspace:5 rspace:5 # up barb left down barb right harpoon
-operator.\u294E.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftRightVector;
+operator.\u294E.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &LeftRightVector;
 operator.\u294F.infix = lspace:5 rspace:5 stretchy direction:vertical # &RightUpDownVector;
-operator.\u2950.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownLeftRightVector;
+operator.\u2950.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &DownLeftRightVector;
 operator.\u2951.infix = lspace:5 rspace:5 stretchy direction:vertical # &LeftUpDownVector;
-operator.\u2952.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftVectorBar;
-operator.\u2953.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightVectorBar;
+operator.\u2952.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &LeftVectorBar;
+operator.\u2953.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &RightVectorBar;
 operator.\u2954.infix = lspace:5 rspace:5 stretchy direction:vertical # &RightUpVectorBar;
 operator.\u2955.infix = lspace:5 rspace:5 stretchy direction:vertical # &RightDownVectorBar;
 operator.\u2956.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownLeftVectorBar;
 operator.\u2957.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownRightVectorBar;
 operator.\u2958.infix = lspace:5 rspace:5 stretchy direction:vertical # &LeftUpVectorBar;
 operator.\u2959.infix = lspace:5 rspace:5 stretchy direction:vertical # &LeftDownVectorBar;
-operator.\u295A.infix = lspace:5 rspace:5 stretchy direction:horizontal # &LeftTeeVector;
-operator.\u295B.infix = lspace:5 rspace:5 stretchy direction:horizontal # &RightTeeVector;
+operator.\u295A.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &LeftTeeVector;
+operator.\u295B.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &RightTeeVector;
 operator.\u295C.infix = lspace:5 rspace:5 stretchy direction:vertical # &RightUpTeeVector;
 operator.\u295D.infix = lspace:5 rspace:5 stretchy direction:vertical # &RightDownTeeVector;
-operator.\u295E.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownLeftTeeVector;
-operator.\u295F.infix = lspace:5 rspace:5 stretchy direction:horizontal # &DownRightTeeVector;
+operator.\u295E.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &DownLeftTeeVector;
+operator.\u295F.infix = lspace:5 rspace:5 stretchy accent direction:horizontal # &DownRightTeeVector;
 operator.\u2960.infix = lspace:5 rspace:5 stretchy direction:vertical # &LeftUpTeeVector;
 operator.\u2961.infix = lspace:5 rspace:5 stretchy direction:vertical # &LeftDownTeeVector;
 operator.\u2962.infix = lspace:5 rspace:5 accent # leftwards harpoon with barb up above leftwards harpoon with barb down
 operator.\u2963.infix = lspace:5 rspace:5 # upwards harpoon with barb left beside upwards harpoon with barb right
 operator.\u2964.infix = lspace:5 rspace:5 accent # rightwards harpoon with barb up above rightwards harpoon with barb down
 operator.\u2965.infix = lspace:5 rspace:5 # downwards harpoon with barb left beside downwards harpoon with barb right
 operator.\u2966.infix = lspace:5 rspace:5 accent # leftwards harpoon with barb up above rightwards harpoon with barb up
 operator.\u2967.infix = lspace:5 rspace:5 accent # leftwards harpoon with barb down above rightwards harpoon with barb down
 operator.\u2968.infix = lspace:5 rspace:5 accent # rightwards harpoon with barb up above leftwards harpoon with barb up
 operator.\u2969.infix = lspace:5 rspace:5 accent # rightwards harpoon with barb down above leftwards harpoon with barb down
 operator.\u296A.infix = lspace:5 rspace:5 accent # leftwards harpoon with barb up above long dash
 operator.\u296B.infix = lspace:5 rspace:5 accent # leftwards harpoon with barb down below long dash
 operator.\u296C.infix = lspace:5 rspace:5 accent # rightwards harpoon with barb up above long dash
 operator.\u296D.infix = lspace:5 rspace:5 accent # rightwards harpoon with barb down below long dash
 operator.\u296E.infix = lspace:5 rspace:5 stretchy direction:vertical # &UpEquilibrium;
 operator.\u296F.infix = lspace:5 rspace:5 stretchy direction:vertical # &ReverseUpEquilibrium;
-operator.\u2970.infix = lspace:5 rspace:5 # &RoundImplies;
+operator.\u2970.infix = lspace:5 rspace:5 accent # &RoundImplies;
 operator.\u2971.infix = lspace:5 rspace:5 accent # equals sign above rightwards arrow
 operator.\u2972.infix = lspace:5 rspace:5 accent # tilde operator above rightwards arrow
 operator.\u2973.infix = lspace:5 rspace:5 accent # leftwards arrow above tilde operator
 operator.\u2974.infix = lspace:5 rspace:5 accent # rightwards arrow above tilde operator
 operator.\u2975.infix = lspace:5 rspace:5 accent # rightwards arrow above almost equal to
 operator.\u2976.infix = lspace:5 rspace:5 accent # less-than above leftwards arrow
 operator.\u2977.infix = lspace:5 rspace:5 accent # leftwards arrow through less-than
 operator.\u2978.infix = lspace:5 rspace:5 accent # greater-than above rightwards arrow
